docs/documentation/concepts/index.md

```diff
@@ -6,29 +6,43 @@ layout: documentation
-If a Channel is linked to an Item, it is "enabled", which means that the functionality that the Item represents is handled through the given Channel.
-Channels can be linked to multiple Items and Items can be linked to multiple Channels.
+## Things, Channels, Items and Links
 
-To illustrate these concepts, take a two-channel actuator that controls two lights:
+**Things** are entities that can be physically added to a system.
+Things may provide more than one function (for example, a Z-Wave multi-sensor may provide a motion detector and also measure room temperature).
+Things do not have to be physical devices; they can also represent a web service or any other manageable source of information and functionality.
 
-![](images/thing-devices-1.png)
+Things expose their capabilities through **Channels**.
+Whether an installation takes advantage of a particular capability reflected by a Channel depends on whether it has been configured to do so.
+When you configure your system, you do not necessarily have to use every capability offered by a Thing.
+You can find out what Channels are available for a Thing by looking at the documentation of the Thing's **Binding**.
+Bindings can be thought of as software adapters, making Things available to your home automation system.
 
-The actuator is the _Thing_. This might be installed in the electrical cabinet, it has a physical address and needs to be setup and configured in order to be used.
-The user is instead interested in the two lights, which are located at different locations in his home. These lights are the desired functionality, thus the _Items_ and they are linked to the _Channels_ of the actuator.
-A _Link_ can be regarded like a physical wire in this example.
+**Items** represent capabilities that can be used by applications, either in user interfaces or in automation logic.
+Items have a **State** and they may receive commands.
 
+The glue between Things and Items are **Links**.
+A Link is an association between exactly one Channel and one Item.
+If a Channel is linked to an Item, it is "enabled", which means that the capability the Item represents is accessible through that Channel.
+Channels may be linked to multiple Items and Items may be linked to multiple Channels.
 
+To illustrate these concepts, consider the example below of a two-channel actuator that controls two lights:
 
+![](images/thing-devices-1.png)
 
+The actuator is a Thing that might be installed in an electrical cabinet.
+It has a physical address and it must be configured in order to be used (remember the physical view introduced at the beginning of this article).
 
+In order for the user to control the two lights, he or she access the capability of the actuator Thing (turning on and off two separate lights) through two Channels, that are Linked to two switch Items presented to the user through a user interface.
```

Comment:
Couldn't be "access the capability of the actuator Thing through two Channels, that are Linked to two switch Items" interpret that he still accesses the channels?

Reply:
I don't think so.  He access the Capability of the Thing **through** the Channel.  Does that help at all?